src/Symfony/Component/Form/Tests/Fixtures/Descriptor/defaults_1.txt

```diff
@@ -0,0 +1,52 @@
+
+[33mBuilt-in form types[39m
+[33m-------------------[39m
+
+ * Symfony\Component\Form\Extension\Core\Type\FormType
```

Comment:
Just wondering: What about removing the namespace for core types when using the txt descriptor? (perhaps mentioning it only once above, in a comment or the section).

Reply:
A long list of short word is also not ideal btw. But for :+1: for short names anyway. Perhaps play a bit with comma separated variants.

Reply:
:+1: Improved core types representation.

Reply:
Nice :)